website/events/templates/events/event.html

```diff
@@ -239,17 +239,35 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                             </td>
                         </tr>
 
-                        {% if event.food_event %}
-                            <tr>
-                                <th>
-                                    Pizza
-                                </th>
-                                <td>
-                                    <a href="{% url "pizzas:index" %}" class="btn btn-primary">
-                                        {% trans "Order" context "pizzas" %}
-                                    </a>
-                                </td>
-                            </tr>
+                        {% if event.food_event %} 
+                            {% if event.food_event == event.food_event.current or event.food_event.just_ended %}
+                                <tr>
+                                    <th>
+                                        Pizza
+                                    </th>
+                                    <td>
+                                        <a href="{% url "pizzas:index" %}" class="btn btn-primary">
+                                            {% trans "Order" context "pizzas" %}
+                                        </a>
+                                    </td>
+                                </tr>
+                            {% elif event.food_event.in_the_future %}
+                                <tr>
+                                    <th>
+                                        Pizza
+                                    </th>
+                                    <td>
+                                        <div>
+                                            <a class="btn btn-primary disabled">
+                                                {% trans "Order" context "pizzas" %}
+                                            </a>
+                                        </div>
+                                        <div class="mt-3">
+                                            <em>Ordering pizza for this event isn't possible yet</em>
```

Comment:
Maybe display when it will be possible to order (food event start)

Reply:
Adding to that: this could be a tooltip

```
<a class="btn btn-primary disabled" data-toggle="tooltip" data-placement="auto" title="{% trans "Ordering pizza for this event will be possible starting {{ event.food_event.start }}." %}">
    {% trans "Order" context "pizzas" %}
</a>
```

Reply:
I like it, I implemented both